ark/docs/src/content/docs/reference/cheat-sheet.md

```diff
@@ -23,6 +23,9 @@ export const currentTsSyntax = type({
 		nested: "string",
 		"optional?": "number"
 	},
+        arrayOfRecord: type({
+                name: "string",
+        }).array(),
```

Comment:
Is this indented more than the others or is it just the way it is displaying?

Reply:
I created the PR ytd from GH, this morning I actually cloned the repo and applied the formatter ;)